docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -193,6 +194,8 @@ Json Parameters:
 -   **Cpuset** - The same as CpusetCpus, but deprecated, please don't use.
 -   **CpusetCpus** - String value containing the cgroups CpusetCpus to use.
 -   **CpusetMems** - Memory nodes (MEMs) in which to allow execution (0-3, 0,1). Only effective on NUMA systems.
+-   **BlkioWeight** - Integer value containing the weight of block IO for container
+      (ie. the relative weight vs other containers).
```

Comment:
I think you should use the same description here "Block IO weight (relative weight...."

To give people some more information, add a link to the "Block IO bandwidth (Blkio) constraints" in Docker Run Reference.